temba/contacts/models.py

```diff
@@ -838,6 +838,19 @@ def get_history(self, after: datetime, before: datetime, include_event_types: se
             .order_by("-created_on")
             .select_related("channel")[:limit]
         )
+        retried_calls = []
+        for call in calls:
+            channel_logs = call.channel_logs.filter(
```

Comment:
Let's add the `.only("is_error", "created_on")` to reduce the data returned on this query, I was a bit concerned about the query being based on the description field since it's not an index, but I believe it's going to be ok since this is querying the channel logs for a specific IVR call which should not be much.